gremlin-driver/src/main/java/org/apache/tinkerpop/gremlin/driver/ser/GraphBinaryMessageSerializerV1.java

```diff
@@ -42,6 +43,7 @@
 
 public class GraphBinaryMessageSerializerV1 extends AbstractMessageSerializer {
     public static final String TOKEN_CUSTOM = "custom";
+    public static final String TOKEN_BUILDER = "builder";
```

Comment:
The new configuration option should be added to the [docs](https://github.com/apache/tinkerpop/blob/TINKERPOP-2150/docs/src/reference/gremlin-applications.asciidoc#graphbinary).

Reply:
Nice catch @dkuppitz , will do!